.circleci/config.yml

```diff
@@ -47,17 +39,11 @@ workflows:
           name: "NixOS 21.11"
           nixpkgs: "21.11"
 
-      # Test against PyPy 2.7
-      - "pypy27-buster":
-          {}
-
-      # Test against Python 3:
-      - "python37":
-          {}
+      # Eventually, test against PyPy 3.8
+      #- "pypy27-buster":
+      #    {}
```

Comment:
Do you suppose a PyPy 3.x configuration will become part of this PR or later on?  One one hand, there was no PyPy 3.x CI so this is not quite a regression.  On the other hand, there was *some* PyPy CI and now there isn't.

I think it would be nice if PyPy 3.x came *soon* if it doesn't come as part of this PR itself.

Reply:
Later PR, I think. https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3697. I'll do that next.